lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1738,6 +1770,27 @@ def test_grade_report(self, persistent_grades_enabled):
                 ignore_other_columns=True,
```

Comment:
Same comment here as [this comment](https://github.com/open-craft/edx-platform/pull/375#discussion_r675504462)